Sources/Paywalls/Components/PaywallImageComponent.swift

```diff
@@ -15,24 +15,24 @@ public extension PaywallComponent {
     struct ImageComponent: PaywallComponentBase {
 
         let type: ComponentType
-        public let url: URL
-        public let cornerRadius: Double
-        public let gradientColors: [ColorHex]
+        public let urlsLid: LocalizationKey
```

Comment:
What does `lid` stand for?

Reply:
Localization ID.

Maybe we should adopt the suffix `LocalizationID`, which is clear.